src/components/gauge/gauge.scss

```
@@ -0,0 +1,222 @@
+@import "../common/_base";
+
+gx-gauge {
+  /**
+  * @prop --component-height:
+   * Set the height or the control. *Must be set before render the control*
+   * (400px by default)
+  */
+  --component-height: 400px;
```

Can we use `height` to set the component's height instead of a custom property?

---

See here: https://github.com/genexuslabs/web-controls-library/commit/0044d2eecc1d975c3dd8191702115ae1a109108f

---

No, we can't. The component needs a height with a specific value to calculate the SVG/Line sizes.
Wht do u have in mind?

---

I think we can because at line 58 the value of this CSS variable is directly assigned to the `height` property.
So it would make perfect sense to have this:

```suggestion
  height: 400px;
```

---

Mmm... set a default `height`, and then change it if somebody wants a different value.
E.g.: 
``` html
   <style>
      gx-gauge.smallGauge{
          height: 200px;
      }
   </style>
   <gx-gauge class="small-gauge">
      ...
   </gx-gauge>
```
I see it as very very useful. A default height if the developer doesn't specify it. And, if the developer set a `height`, the control uses that height.
I ran that approach with the custom CSS var... but it will be better if we can remove complexity!
That is what are you suggesting?

---

I apply your suggestion in commit c2e09ed